modular_skyrat/modules/cryosleep/code/admin.dm

```diff
@@ -0,0 +1,22 @@
+// Send player in not-quiet cryopod. If with_paper = TRUE, place a paper with notification under player.
```

Comment:
It needs to be `///` and not `//` for it to count as documentation for the autodoc parser :)
```suggestion
/// Send player in not-quiet cryopod. If with_paper = TRUE, place a paper with notification under player.
```